internal/compile/compile.go

```diff
@@ -39,7 +39,11 @@ import (
 	"go.starlark.net/syntax"
 )
 
-const debug = false // TODO(adonovan): use a bitmap of options; and regexp to match files
+// Disassemble causes the assembly code for each function
+// to be printed to stderr as it is generated.
+var Disassemble = true
```

Comment:
Absolutely it should be false by default. I think you must have looked at the short-lived first draft of this PR, which I quickly updated once I noticed the problem. (I suspect you must have reviewed this CL promptly but forgot to hit 'send'; if not, I wonder how this could have happened.)

Given that its default is now false, embedders won't be able to _enable_ it, but I think that's ok for now; they can always hack the flag to true if they need to debug something. In other words, this is feature only ofcmd/starlark or the REPL feature for now. If there is a real need, we can add a starlark.SetDisassembly(bool) function to the API (but we can't move the flag itself as 'compile' cannot depend on 'starlark'.)

Reply:
Ah, that's exactly what happened. I remember typing this while distracted at the summit.